releasenotes/notes/40784.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+  - 40784
+releaseNotes:
+  - |
+    **Added** contains matching to Authorization Policy HeaderMatcher
```

Comment:
The PR describes that this PR changes existing behavior: "This change will replace the previous behavior of headers being bounded with \* only matching on the suffix."

The release notes should warn the changing behavior.

Reply:
Updated the release notes.